src/pages/search/components/NoSupport.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import FontAwesome from 'react-fontawesome';
+import { Row, Col } from 'react-bootstrap';
+
+function NoSupport() {
+  return (
+    <div className='flexWrapper'>
```

Comment:
This uses `.flexWrapper` and `.verticalAlign` from `NoResults/404`. Should we move those classes to global or some section specific to these kind of error pages?

Reply:
Probably, but I think it is out of scope for this PR. I noticed the latest version of Gatsby comes with a CSS Modules plugin, so I would like to give that a go if I can get the latest version of Gatsby to work here

Reply:
And I think it will be more like a CSS utils file, so `.flexSpaceAround` could go in it too